src/pages/umi/guides/serializing-and-deserializing-transactions.md

```diff
@@ -1,57 +1,294 @@
 ---
-title: Recipes
-metaTitle: Umi - Recipes
-description: How-to guides for Metaplex's Umi product.
+title: Serializing, Deserializing, and sending Transactions
+metaTitle: Umi - Serializing, Deserializing, and sending Transactions
+description: Learn how to Serialize and Deserialize Transactions to move them across different environments while using the Metaplex Umi client.
+created: '08-2-2024'
+updated: '08-2-2024'
 ---
 
-## Passing Transactions from Backend to Frontend (and vice versa)
+**In this guide we're going to talk about:**
+- Serializing and Deserializing Transaction
+- Noop Signer
+- Partially signed transactions
+- Passing transactions from different environments
 
-When passing a transaction from backend to Frontend or vice versa you will need to serialize the transaction in order to transport it between locations. This can also be performed the same way in reverse.
+## Introduction
 
-#### Backend
+Transaction are usually serialized to facilitate movement across different environments. But the reasons could be different:
+- You may require signatures from different authorities stored in separate environments.
+- You may wish to create a transaction on the frontend, but then send and validate it on the backend before storing it to a database.
+
+For example, when creating NFTs, you may need to sign the transaction with the `Collection Authority` keypair to authorize the NFT into the Collection. To safely sign it, without exposing your keypair, you could first create the transaction client side/frontend, partially sign the transaction with the purchaser, serialize the transaction and then send it to your secure environment (server/backend). You can then safely deserialize the transactions and sign it with the `Collection Authority` keypair without exposing the keypair in a non secure environment.
```

Comment:
We should propose to do it in the other direction. Build it in the backend, send to frontend for user to sign.

Otherwise we would recommend to introduce a security risk where they sign a TX in the backend that would have to be thoroughly verified to be just a mint tx and not, for example, a drain

Reply:
for sake of completeness it might also make sense to mention that this collection authority signature in a backend is not needed when using CM?